x-pack/plugins/security_solution/public/common/components/event_details/summary_view.tsx

```diff
@@ -0,0 +1,167 @@
+  { id: 'signal.rule.risk_score', label: ALERTS_HEADERS_RISK_SCORE },
+  { id: 'host.name' },
+  { id: 'user.name' },
+  { id: SOURCE_IP_FIELD_NAME, fieldType: IP_FIELD_TYPE },
+  { id: DESTINATION_IP_FIELD_NAME, fieldType: IP_FIELD_TYPE },
+];
+
+const getDescription = ({
+  contextId,
+  eventId,
+  fieldName,
+  value,
+  fieldType = '',
+  linkValue,
+}: {
+  contextId: string;
+  eventId: string;
+  fieldName: string;
+  value?: string | null;
+  fieldType?: string;
+  linkValue?: string;
+}) => {
+  if (fieldName === 'signal.status') {
+    return (
+      <EuiBadge>
+        <FormattedFieldValue
+          contextId={`alert-details-value-formatted-field-value-${contextId}-${eventId}-${fieldName}-${value}`}
+          eventId={eventId}
+          fieldName={fieldName}
+          fieldType={fieldType}
+          value={value}
+        />
+      </EuiBadge>
+    );
+  }
+
+  return (
+    <FormattedFieldValue
+      contextId={`alert-details-value-formatted-field-value-${contextId}-${eventId}-${fieldName}-${value}`}
+      eventId={eventId}
+      fieldName={fieldName}
+      fieldType={fieldType}
+      value={value}
+      linkValue={linkValue}
+    />
+  );
+};
+
+export const SummaryViewComponent: React.FC<{
+  browserFields: BrowserFields;
+  data: TimelineEventsDetailsItem[];
+  eventId: string;
+  timelineId: string;
+}> = ({ data, eventId, timelineId, browserFields }) => {
+  const [investigationGuide, setInvestigationGuide] = useState<string | null>(null);
+
+  const ruleIdField = useMemo(() => data.find((d) => d.field === 'signal.rule.id'), [data]);
+  const ruleId = getOr(null, 'originalValue', ruleIdField);
+  const { rule: maybeRule } = useRuleAsync(ruleId);
+  const summaryList = useMemo(() => {
+    return data != null
+      ? fields.reduce<Summary>((acc, item) => {
+          const field = data.find((d) => d.field === item.id);
+          if (!field) {
+            return acc;
+          }
+          const linkValueField =
+            item.linkField != null && data.find((d) => d.field === item.linkField);
+          const linkValue = getOr(null, 'originalValue', linkValueField);
+          const value = getOr(null, 'originalValue', field);
+          const category = field.category;
+          const fieldType = get(`${category}.fields.${field.field}.type`, browserFields) as string;
+          const description = getDescription({
+            contextId: timelineId,
+            eventId,
+            fieldName: item.id,
+            value,
+            fieldType: item.fieldType ?? fieldType,
+            linkValue: linkValue ?? undefined,
+          });
+
+          return [
+            ...acc,
+            {
+              title: item.label ?? item.id,
+              description,
+            },
+          ];
+        }, [])
+      : [];
+  }, [browserFields, data, eventId, timelineId]);
+
+  useEffect(() => {
+    if (maybeRule != null && maybeRule.note != null) {
+      setInvestigationGuide(maybeRule.note);
```

Comment:
do we need an `useEffect` for that? maybe using inline `rule.note` instead would do?